src/hardware/pslab/api.c

```diff
@@ -67,21 +67,31 @@ static GSList *scan(struct sr_dev_driver *di, GSList *options)
 
 	for (l = device_paths; l; l = l->next)
 	{
-		if (path && path != l->data) {
+	    char *device_path = l->data;
```

Comment:
You seem to be mixing tabs and spaces for indentation. Tab indentation seems to be the standard in libsigrok, so stick to that.

Reply:
I have made that change in my IDE formatter, not sure how this still happened. Will try and keep an eye out for this. Updated the PR.